### PR TITLE
Estimates chapter

### PR DESCRIPTION
**Pull request by tungfam**
Hi team 👋,

Please take some time to review the Estimates chapter of the cookbook.

Feel **very** free to comment if you think some estimates in the cheat sheet can be adjusted/edited. Or if there are some other common features that we can add to cheat sheet.

Since the topic is very broad and can have many tips/advice feel free to suggest what can be added and I will add them.

⛳️

### CODE REVIEW

File: 6-estimates.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+### Gather requirements and inputs
+
+#### 1. Get to know your client and project 💼.
+a. make a quick research or ask the sales representative about the company, their business, missions (read why in [Communication Tips](#communicationtips) section); 
+b. their audience since it might be people who may require Accessibility features in the app;
+c. review information about the project;
+
+#### 2. Get features list 📜.
+The features can be one or more from the list:
+1. Visual
```
Comment (EvgenyMatviyenko) [inline 6-estimates.md]: Seems like first one point should be `Written` and second one - `Visual`

File: 6-estimates.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+Image picker (default) | 2 | 4 | n/a
+Image picker (custom) | 16 | 24 | n/a
+Image cropper ([lib](https://github.com/ruslanskorb/RSKImageCropper)) | 4 | 8 | n/a
+Image cropper (custom) | 24 | 40 | n/a
+Internatialization | 8 | 16 | can be more if it's ؁ؓؑ  ؐ  ؕ  (arabic)
+Photo Camera (custom) | 24 | 40 | capture, focus, zoom
+Video Camera (custom) | 40 | 60 | simple video shooting
+Video Camera (lib) | 16 | 32 | simple video shooting
+Video Player (custom) | 24 | 32 | for a player with controls like default has
+Simple networking using our [lib](https://github.com/uptechteam/Net) | 4 | 8 | n/a
```
Comment (arthur-here) [inline 6-estimates.md]: Let's change the link from our lib to the `Moya` https://github.com/Moya/Moya. Last meeting we decided to stick to the Moya as our networking solution
Multipart requests can also be handled by Moya

File: 6-estimates.md (hunk 1 above)
Comment (tungfam) [reply 6-estimates.md]: oh yes:) good catch! thanks

**Comment by tungfam** [comment]
@arthur-here will add, thanks!

huh maybe it's a sign to convert it to an article:))

File: 6-estimates.md (hunk 2 above)
Comment (tungfam) [reply 6-estimates.md]: got it, okay. thanks.

File: 6-estimates.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+More examples [here](https://github.com/stanfy/ios-components-bikeshedding)
+
+### Special cases
+
+**We do include supporting dev services:**
+- CI setup
+- Crashlytics
+- Analytics services
+ 
+**Consider languages.**
```
Comment (R0manUp) [inline 6-estimates.md]: Maybe `Consider Localization` will be more proper naming

Comment (tungfam) [reply 6-estimates.md]: Agree. thanks!

**Comment by tungfam** [comment]
@aLHF hm, valid point. it's similar case like when 2 people are making estimates, it's advised for them not to look into each other's estimates so that we don't mislead each other.

but  the solution can be:
1. The person may not check the cheat sheet and do the estimate on their own.
2. For cases when the person had no experience with a certain feature, it will be useful to use the sheet to give the more accurate estimate.

Do you think having these 2 solutions may help us?

**Comment by arthur-here** [comment]
@aLHF thank you for your valuable feedback!
I would like to clarify two points:
1) This cheat sheet should be used only for the *Rough Estimates*. Rough Estimate is done in 1-2 hours and such a cheat sheet is a must have for that.
2) There is no place for personal skills in Rough Estimates, as at the time of the estimate it's not yet known who would work on the project.

So based on that, I would leave the Cheat Sheet section in the chapter, but would clarify that this is "Rough Estimates Cheat Sheet"

**Comment by aLHF** [comment]
@tungfam, @arthur-here if it's **rough** estimates then i totally support it.

File: 6-estimates.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+![](https://github.com/uptechteam/ios-cookbook/blob/feature/estimates/resources/estimates_screenshots/login.png)
+
+can be broken into the following parts:
+1. Login using credentials
+2. Facebook login
+3. Additional Facebook integration (optional depending on existence of other facebook related features)
+
+Note: "Forgot password" feature will most likely have a separate screen for it.
+
+Sometimes it's not possible to break down a relatively big feature, for example implementing a chat or video streaming. In this case the feature be picked as is.
```
Comment (tolgaarikan) [inline 6-estimates.md]: Maybe we can add `can` here. In this case the feature `can` be picked as is.

File: 6-estimates.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+Finally, after breaking down to smaller features, choose one feature and move the next step.
```
Comment (tolgaarikan) [inline 6-estimates.md]: move `to` the next step.

**Pull request merged**